bin/spring

```diff
@@ -4,12 +4,12 @@
 # It gets overwritten when you run the `spring binstub` command.
 
 unless defined?(Spring)
-  require "rubygems"
-  require "bundler"
+  require 'rubygems'
```

Comment:
Huh, actually, those are auto-generated files, which is why my rubocop settings weren't at issue.